students/meeryu/users/views.py

```diff
@@ -44,7 +50,8 @@ def post(self,request):
                 
                 if not User.objects.filter(email = email, password = password):
                     return JsonResponse({"message" : "INVALID_USER"}, status = 401)
-                return JsonResponse({"message" : "SUCCESS"}, status=201)
+
+                return JsonResponse({"message" : "SUCCESS"}, status = 201)
```

Comment:
지금보니 상태코드가 잘못 작성되어 있엇네요. 201은 리소스의 생성을 의미하는 코드인데 ㅎㅎ 200으로 수정해주세요!
```suggestion
                return JsonResponse({"message" : "SUCCESS"}, status = 200)
```